components/content/repulsion-effect/index.vue

```diff
@@ -0,0 +1,133 @@
+<script setup lang="ts">
+import { TresCanvas, useTresContext } from '@tresjs/core'
+import { PCFSoftShadowMap, BasicShadowMap, PCFShadowMap, NoColorSpace, LinearSRGBColorSpace, SRGBColorSpace, NoToneMapping } from 'three'
+import { OrbitControls, Box, vAlwaysLookAt, vLightHelper } from '@tresjs/cientos';
+
+const rectAreaLightRef = ref(null)
+
+const gl = {
+  alpha: true,
+  shadows: true,
+  shadowMapType: PCFSoftShadowMap,
+  powerPreference: "high-performance",
+}
+
+
+watchEffect(() => {
+  if (rectAreaLightRef.value) {
+    rectAreaLightRef.value.lookAt(0, 0, 0);
```

Comment:
Just as a question, if you add the lookAt at the component it doesn't work?

Like <TresRectAreaLight ref="rectAreaLightRef" color="#341212" :decay="0" :width="1000" :height="1000"
      :position="[5, 20, 50]" :look-at="[0,0,0]" />

Reply:
You are right, I didn't actually know that you could use the `look-at` like that.